scripts/cleanmanpages.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# -- cleans the unnecessary directory reference manpages _*.3
+
+BASE=$1
+
+rm -Rf $BASE/_*.3
```

Comment:
If this is under `build/`, we already `.gitignore` it; if `.3` files are placed somewhere else in the tree, we might have to add `*.3` to `.gitignore`.

Otherwise looks good.

Reply:
Its falls under `doxygen`.  Maybe we add a `.gitignore` with `.3` in that directory as well